### ISSUE TITLE
MAC OS Sierra update

### ISSUE THREAD

**Issue by romanlelek** [addyosmani/grunt-uncss#194]
Hello, I have an issue - connected with Sierra update - before the update my settings worked perfectly, no problem, after update the same configuration stopped working anymore. But strange thing is that I cant see any error - when I run uncss task, it says Finished, but nothing is created in dist folder as before - the task doesnt create any output file.

My settings:

Grunt-uncss task:
```
module.exports = function(grunt) {
    return {
        dist: {
            files: [
                { src: '<%= paths.dist %>/all_transactional.html', dest: '<%= paths.dist %>/css/nomediaq.css' }
                ,{ src: '<%= paths.dist %>/testing_template.html', dest: '<%= paths.dist %>/css/testing_template.css' }
            ]
        },
        options: {
            compress: true,
            ignore: ['.ExternalClass', '.ExternalClass p', '.ExternalClass span', '.ExternalClass font', '.ExternalClass td', '.ExternalClass div', 'a[href^=tel]',
                'a[href^=sms]', 'a[x-apple-data-detectors]', 'a:hover', 'code', 'pre', '.btn_hoover', '.menuContent a', '.yshortcuts', '.ReadMsgBody'
            ]
        }
    }
}

```

Gruntfile:

```
module.exports = function(grunt) {

    require('jit-grunt')(grunt);
    require('load-grunt-config')(grunt, {

        // Pass data to tasks
        data: {

            // Re-usable filesystem path variables
            paths: {
                dist: 'dist'
            }
        }
    });
};

```

Logging when I run it with grunt uncss --debug --verbose

> Running "uncss:dist" (uncss) task
> [D] Task source: ..../node_modules/grunt-uncss/tasks/uncss.js
> Verifying property uncss.dist exists in config...OK
> Files: dist/all_transactional.html -> dist/css/nomediaq.css
> Files: dist/testing_template.html -> dist/css/testing_template.css
> Options: report="gzip", compress

Anyone any idea what should I change? Thank you.

Best regards,
Roman

**Comment by mikelambert** [comment]
You probably need to share the relevant bits of your grunt config. (And fixing the formatting above would be nice!)

**Comment by romanlelek** [comment]
Ok, no problem, I have added my grunt file too. Little background - in my dist folder I have already created HTML file (in my case e-mail), which is called "all_transactional.html" and from uncss I want to remove not used css (e.g. obsolete media queries etc.) and create a file in dist/css/nomediaq.css then I will inline the css and will continue to code my e-mails....

Is there any chance to see whats really happing inside the running task? e.g. to turn on some logging to see where is the problem? The result is not saying anything - just that the task finished:

> (Running "uncss:dist" (uncss) task
> [Finished in 65.4s]

. No info about why there is no file created in dist/css...no error message...nothing and I am asking why?

**Comment by mikelambert** [comment]
I cannot see where grunt-uncss is actually used/called in the above two configs.

**Comment by romanlelek** [comment]
Well, it is called through this plugin load-grunt-config (e.g. this row require('load-grunt-config')) - it looks for aliases which I have defined in my file aliases.yaml. 
I have just ran grunt uncss --debug --verbose

(shorted version)

```
Command-line options: --verbose, --debug=1
Reading "Gruntfile.js" Gruntfile...OK
Registering Gruntfile tasks.
Initializing config...OK
Registering "grunt-uncss" local Npm module tasks.
Reading .../node_modules/grunt-uncss/package.json...OK
Parsing .../node_modules/grunt-uncss/package.json...OK
Loading "uncss.js" tasks...OK
+ uncss
Loading "Gruntfile.js" tasks...OK
```

The grunt-uncss is succesfully loaded (registered), but when I run it, there is no output file. What is the role of PhantomJS there? Is there any for my case? I cant see any system process running PhantomJS.

From my beginner point of view it looks like that there is some uncaught error or exception somewhere around this lines of code (e.g. 48 - 53):

```
grunt.file.write( file.dest, output );
                    grunt.log.writeln('File ' + chalk.cyan( file.dest ) + ' created: ' + maxmin( report.original, output, options.report === 'gzip' ) );
                    if (typeof(options.reportFile) !== 'undefined' && options.reportFile.length > 0) {
                        grunt.file.write(options.reportFile, JSON.stringify(report));
                    }
                    done();
```
Because when I run it, it doesn't throw any error to be thrown, but there is no row in log for this row:

```
grunt.log.writeln('File ' + chalk.cyan( file.dest ) + ' created: ' + maxmin( report.original, output, options.report === 'gzip' ) );
```

**Comment by romanlelek** [comment]
No idea anyone, so I have just solved it by myself. 

I have noticed that I was using older version than current. The problem was fixed by updating to current 0.6.1

**Comment by dr-martin** [comment]
I have the same problem:
In my case (maybe the same situation): PhantomJS PATH is the interruption, the real solution is not the update but with the update the PhahtomJS is re-configurated.
